packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -33,6 +33,7 @@ import type { InsertableContent } from "./toMapTree.js";
 import { isLazy, type FlexListToUnion, type LazyItem } from "./flexList.js";
 import { LeafNodeSchema } from "./leafNodeSchema.js";
 import { TreeNodeValid } from "./treeNodeValid.js";
+import type { SimpleFieldSchema } from "./api/index.js";
```

Comment:
this probably should be imported here. SimpleSchema probably should move out of api.